protos/feast/serving/ServingService.proto

```diff
@@ -20,7 +20,6 @@ package feast.serving;
 
 import "google/protobuf/timestamp.proto";
 import "feast/types/Value.proto";
-import "tensorflow_metadata/proto/v0/statistics.proto";
```

Comment:
I think we can update `Makefile` and `setup.py` to remove references to the tensorflow protos. Can we also remove the protos themselves? I was just trying to do that in #2058

Reply:
Ah, I saw your PR and I thought you would merge it)
Yeah, we may delete statistics, since there's no need in CoreService either

Reply:
I wasn't convinced that we'd be killing CoreService so I punted the PR ;) I can revive it if that's the direction we're taking.

Reply:
FYI CoreService.proto is deleted in this PR

Reply:
In that case I'll revive my PR and rebase off of this.